Main.java

```diff
@@ -0,0 +1,31 @@
+import java.util.HashMap;
+
+public class Main {
+    public static void main(String[] args) {
+        //Task 1.
+        HashMap<String, String> FirstHashMap = new HashMap<String, String>();
+        FirstHashMap.put("978-1-60309-452-8", "A letter to Jo");
+        FirstHashMap.put("978-1-60309-459-7", "Lupus");
+        FirstHashMap.put("978-1-60309-444-3", "Red Panda and Moon Bear");
+        FirstHashMap.put("978-1-60309-461-0", "The Lab");
```

Comment:
can be placed in function list too but use some initialized data

Reply:
07301715

Reply:
good, we have a menu, but we mentioned that you have to work with user inputs not predefinied strings! also will be better if the menu in a while loop (input q for exit)

Reply:
86ecb748

Reply:
5d3bf733